panel/src/components/Forms/FormButtons.vue

```diff
@@ -121,10 +118,10 @@ export default {
 			return null;
 		},
 		lockState() {
-			return this.supportsLocking && this.lock ? this.lock.state : null;
+			return this.supportsLocking && this.$content.lock?.state;
 		},
 		supportsLocking() {
-			return this.lock !== false;
+			return this.$content.lock !== false;
```

Comment:
`Suggestion`: It might be cool to have `this.$content.supportsLocking` as well.

Reply:
I left this out as I right now I am doubtful we will need it in the future. My thoughts right now are that locking would completely be dealt with on the backend (if it's not supported, not done). And the Panel receives just the info whether something is locked. If locking isn't supported, that info will just always be empty.

Reply:
Good point. We can already hand it over as a view prop